src/host/screenInfo.cpp

```diff
@@ -1950,6 +1950,130 @@ bool SCREEN_INFORMATION::_IsInVTMode() const
     return WI_IsFlagSet(OutputMode, ENABLE_VIRTUAL_TERMINAL_PROCESSING);
 }
 
+// Routine Description:
+// - Sets a VT tab stop in the column sColumn. If there is already a tab there, it does nothing.
+// Parameters:
+// - sColumn: the column to add a tab stop to.
+// Return value:
+// - none
+// Note: may throw exception on allocation error
+void SCREEN_INFORMATION::AddTabStop(const SHORT sColumn)

```

Comment:
there may have been a merge conflict -- this reintroduces the tab stop functions that were deleted in master